Extensions/CGUnit.js

```diff
@@ -7,7 +7,7 @@ Object.defineProperties(wow.CGUnit.prototype, {
         aura.name === name &&
         aura.casterGuid &&
         me.guid &&
-        me.equals(aura.casterGuid)
+        me.guid.equals(aura.casterGuid)
```

Comment:
`me.guid && me.guid.equals(aura.casterGuid)`
can be simplified to
`me.equals(aura.casterGuid)`

Reply:
I had it originally, but it wasn't working. I will try again.

Reply:
Yeah, doesn't work if I don't use me.guid to compare left hand side.